### PR TITLE
draft for storing issue briefings

### PR DESCRIPTION
**Pull request by Badsauce**
GUI Editor
image

API Return value `/issue-briefings`
image

### CODE REVIEW

**Comment by Badsauce** [comment]
@kevincianfarini

File: app/api/issue-briefing/config/routes.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "policies": []
+      }
+    },
+    {
+      "method": "GET",
+      "path": "/issue-briefings/count",
+      "handler": "issue-briefing.count",
+      "config": {
+        "policies": []
+      }
```
Comment (kevincianfarini) [inline app/api/issue-briefing/config/routes.json:17]: What's the rationale for having a `count` of the issue briefings?

File: app/api/issue-briefing/models/issue-briefing.settings.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      "type": "richtext"
+    },
+    "sample_social_media_post": {
+      "type": "richtext",
+      "required": true
+    },
+    "additional_policy_resources": {
+      "type": "richtext"
+    }
+  }
```
Comment (kevincianfarini) [inline app/api/issue-briefing/models/issue-briefing.settings.json:52]: Can you explain to me what these `attributes` mean? Are these supposed to correspond to columns in the DB?

File: app/api/issue-fact/config/routes.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+  "routes": [
+    {
+      "method": "GET",
+      "path": "/issue-facts",
+      "handler": "issue-fact.find",
+      "config": {
+        "policies": []
+      }
+    },
```
Comment (kevincianfarini) [inline app/api/issue-fact/config/routes.json:10]: nit: what's an issue fact? Is that the same thing as a talking point?

**Review by kevincianfarini — APPROVED** [review]
Had a few questions, but mostly it looks good to me.

**Pull request closed without merging**